src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/SpringQuartzConfig.java

```diff
@@ -0,0 +1,32 @@
+package io.skai.okta.internshipstreamingserviceparserxml.scheduler;
+
+import org.quartz.JobBuilder;
+import org.quartz.JobDetail;
+import org.quartz.Trigger;
+import org.quartz.TriggerBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import static org.quartz.SimpleScheduleBuilder.simpleSchedule;
+
+@Configuration
+public class SpringQuartzConfig {
+    @Bean
+    public JobDetail jobDetail() {
+        return JobBuilder.newJob().ofType(CheckUpdateJob.class)
+                         .storeDurably()
+                         .withIdentity("Updater")
```

Comment:
use job key for identity,

example https://github.com/kenshoo/fb-product-fetcher/blob/master/application/src/main/java/io/skai/fbproductfetcher/quartz/config/QuartzJobConfig.java#L19

Reply:
done